spec/system/management/account_spec.rb

```diff
@@ -102,4 +102,26 @@
     expect(page).to have_css("a[href='javascript:window.print();']", text: "Print password")
     expect(page).to have_css("div.for-print-only", text: "another_new_password", visible: :hidden)
   end
+
+  describe "When a user has not been selected" do
+    before do
+      Setting["feature.user.skip_verification"] = "true"
+    end
+
+    scenario "we can not reset password via email" do
+      visit management_root_path
+
+      click_link "Reset password via email"
+
+      expect(page).to have_content "To perform this action you must select a user"
```

Comment:
How about checking we've been redirected?

```suggestion
      expect(page).to have_content "To perform this action you must select a user"
      expect(page).to have_current_path ...
```

Same in the other tests.